### DISCUSSION TITLE
Getting inappropriate communications from users of this program

### DISCUSSION THREAD

**Discussion by jleatherman3** [simple-login/app#876] [Q&A]
I am getting spammed daily by users of simplelogin.io to mask their email to send me porn and propositions to meet them in person for inappropriate things. They use different names/emails everyday saying this is anna or jessica ect and send nude pictures in the emails and when I click the email it has random characters and @simplelogin.io so I conclude they are members of this group sending this out. 

I would like admins to contact me as quickly as possible to address this and stop these messages from being sent to me. I have emailed the support address as well. Please terminate all accounts involved using their alias to send out this type of email as it is inappropriate, unwanted, and i am sure violated the terms of service of your site. 

I would suggest a filter be put up to scan emails containing images for porn and deny them being sent out in the future.

I would appreciate the urgent attention to this matter.

**Comment by nguyenkims** [comment]
@jleatherman3 sorry for the trouble! Can you send us the following info about the abuses to hi[at]simplelogin.io ?

- the addresses that send you these emails
- one copy of these emails
- date and time when that issue happened.
